### PR TITLE
Feat/user auth

### PR DESCRIPTION
**Pull request by shetkar1910**
Please check my code for emp_list file.

### CODE REVIEW

File: emp_list.html
Comment (rup-m) [inline emp_list.html:1]: acceptable changes! good job

File: login.html [hunk 1]
```diff
@@ -2,6 +2,10 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>GitHub test</title>
+=======
```
Comment (rup-m) [inline login.html:8]: can we have another name for meta name?

File: emp_list.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+
```
Comment (Ojaswini08) [inline emp_list.html:8]: Could you please add Docstring

**Review by shetkar1910 — COMMENTED** [review]
hello , please check my code

File: login.html [hunk 3]
```diff
@@ -17,6 +21,9 @@ <h1>Welcome to the Login page </h1>
 
     <p>this part is directly added on Github</p>
     <h2>here I want to make changes on VScode</h2>
+    <h2>Neew line is added.</h2>
```
Comment (rup-m) [inline login.html:24]: please check spelling.

**Pull request merged**